starter/js/atm.js

```diff
@@ -1,43 +1,54 @@
 //Begin with the document ready function
+$(document).ready(function(){
 
+}
     //Checking account deposit function
-
       //On click of the depositChecking button
-
+$("#depositChecking").click(function(event){
         //Get value from the amountChecking input field
-
+   var amountBalance = $("#amountChecking").val();
         //Take that value and add it to the existing value in the checkingBalance div
-
+   var checkingBalance = amountBalance.val();
     //Checking account withdrawl funtion
-
+});
+event();
```

Comment:
What is this? It seems to be out of place..? You don't need either of these free standing `event();` s